### ISSUE TITLE
How can I show accuracy, precision, recall, f1-measure of this model?

### ISSUE THREAD

**Issue by Faisal-95** [susanli2016/Machine-Learning-with-Python#27]

**Comment by jusgowiturs** [comment]
Use 
from sklearn.metrics import classification_report

To evaluate model
